torch/csrc/distributed/rpc/utils.cpp

```diff
@@ -42,7 +43,10 @@ std::unique_ptr<RpcCommandBase> deserializeRequest(const Message& request) {
       return RRefForkRequest::fromMessage(request);
     }
     case MessageType::MESSAGE_WITH_AUTOGRAD_REQ: {
-      return RpcWithAutograd::fromMessage(request);
+      return autograd::RpcWithAutograd::fromMessage(request);
+    }
+    case MessageType::PROPAGATE_GRADIENTS_REQ: {
```

Comment:
`MESSAGE_WITH_AUTOGRAD_REQ` and `PROPAGATE_GRADIENTS_REQ` do not look that related. How about `FORWARD_AUTOGRAD_REQ` and `BACKWARD_AUTOGRAD_REQ`? (and for resp as well)